presto-docs/src/main/sphinx/functions/geospatial.rst

```diff
@@ -58,6 +58,17 @@ Constructors
 
     Returns a geometry type polygon object from WKT representation.
 
+.. function:: ST_SphericalGeometryFromGeometry(Geometry) -> SphericalGeometry
+
+    The SphericalGeometry type supports the same set of shapes as Geometry, e.g. point, multipoint, linestring, multilinestring, polygon, multipolygon, etc., but assumes they're
```

Comment:
perhaps, `Converts a geometry value defined in (latitude, longitude) space to a geometry on a sphere of Earth's radius.`